db/view/view.cc

```diff
@@ -2097,7 +2097,7 @@ future<> view_builder::drain() {
 
 future<> view_builder::stop() {
     vlogger.info("Stopping view builder");
-    return drain();
+    co_return;
 }
```

Comment:
Nit: put an assertion here which checks that `drain()` was indeed run (it needs to be run in order for the view builder to be properly stopped) - for example, you could check if `_as.abort_requested()` is true